eng/scripts/Language-Settings.ps1

```diff
@@ -183,13 +189,13 @@ function Update-dotnet-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$
 
 
 # function is used to auto generate API View
-function Find-dotnet-Artifacts-For-Apireview($artifactDir, $pkgName)
+function Find-dotnet-Artifacts-For-Apireview($artifactDir)
```

Comment:
It might still be worth passing the package name to potentially verify. Also keep in mind that the signature of this method should be the same across all the languages so if other languages need this then we should keep it here as well.